### DISCUSSION TITLE
NIghtly Builds

### DISCUSSION THREAD

**Discussion by dlim04** [kraxarn/spotify-qt#276] [Q&A] [answered]
Hello, I really like the project. Just started using spotify-qt but will probably be using it every day from now on after having to decide between either deleting the CEF Spotify client or closing some Chrome tabs.

Are you still doing Nightly Builds? [spotify-qt-nightly](https://kraxarn.github.io/spotify-qt-nightly/) doesn't appear to have a new build since August. I ended up downloading the version built by the latest CI test as I didn't want to set up the tooling for building from source on my host machine. It might be an idea just to make the artifacts from the tests more accessible if you don't want to maintain the nigtlies as there are some features (especially the librespot token auth feature) that make the application usable for me that haven't been released.

Further to that, do you have a release planned soon?

Thanks for the really cool project and if I have time in the future I'd love to make some contributions.

**Comment by kraxarn** [comment, marked as answer]
Oops, thanks for pointing it out! 😅 
GitHub just disabled the builds without me realising, but they should now be available as usual again!

Also, yes, with all the recent important changes to librespot and the Spotify API, I do plan a release within the next few days.
Until then, the nightly builds are basically RC builds.
